### PR TITLE
Support GTP-U PDU Session Container

### PR DESCRIPTION
**Pull request by TetsuyaMurakami**
This is just a checklist to guide you. You can remove it safely.

**Checklist:**

-   [X] If you are new to Scapy: I have checked <https://github.com/secdev/scapy/blob/master/CONTRIBUTING.md> (esp. section submitting-pull-requests)
-   [X] I squashed commits belonging together
-   [X] I added unit tests or explained why they are not relevant
-   [X] I executed the regression tests for Python2 and Python3 (using `tox` or, `cd test && ./run_tests_py2, cd test && ./run_tests_py3`)

> brief description what this PR will do, e.g. fixes broken dissection of XXX
This PR can support GTP-U PDU Session Container in contrib/gtp.py which is used for 5G service.

> if required - short explanation why you fixed something in a way that may look more complicated as it actually is
No fix. This is for the new feature support for GTP-U.

> if required - outline impacts on other parts of the library
Nope.

fixes #xxx (add issue number here if appropriate, else remove this line)
None

### CODE REVIEW

File: test/contrib/gtp.uts [hunk 1]
```diff
@@ -16,6 +16,25 @@ assert a.E == a.S == a.PN == 0
 a = GTPHeader(raw(GTP_U_Header()/GTPErrorIndication()))
 assert isinstance(a, GTP_U_Header)
 
+a.show(dump=True)
```
Comment (guedou) [inline test/contrib/gtp.uts]: Could you use the same kind of tests done later in the file?
- dissection: https://github.com/secdev/scapy/blob/master/test/contrib/gtp.uts#L57
- instanciation: https://github.com/secdev/scapy/blob/master/test/contrib/gtp.uts#L73

Comment (TetsuyaMurakami) [reply test/contrib/gtp.uts]: I modified the test for gtp pdu session container. I removed show and hexdump to check the header. Instead, I added some assert to check if the header is correct or not.

**Comment by guedou** [comment]
You have PEP08 related errors, see https://travis-ci.com/secdev/scapy/jobs/242456635#L660. Locally you can easily reproduce them by installing the `tox` tool and invoking `tox -e flake8`.

**Comment by TetsuyaMurakami** [comment]
Fix PEP08 related issue.

File: scapy/contrib/gtp.py [hunk 2]
```diff
@@ -285,6 +291,46 @@ def guess_payload_class(self, payload):
...
 
+class GTPPDUSessionContainer(Packet):
+    name = "GTP PDU Session Container"
+    fields_desc = [ByteField("ExtHdrLen", None),
+                   BitField("type", 0, 4),
+                   BitField("spare1", 0, 4),
+                   BitField("P", 0, 1),
+                   BitField("R", 0, 1),
+                   BitField("QFI", 0, 6),
+                   ConditionalField(XBitField("PPI", 0, 3), lambda pkt: pkt.P == 1),  # noqa: E501
```
Comment (guedou) [inline scapy/contrib/gtp.py]: We do not accept `noqa` for new code.

Comment (TetsuyaMurakami) [reply scapy/contrib/gtp.py]: Thank you for your input. I removed noqa and re-run pep08 check. The pep08 check looks fine now after removing noqa.

Comment (guedou) [reply scapy/contrib/gtp.py]: Thanks!

**Pull request merged**